pydis_site/apps/api/migrations/0046_reminder_jump_url.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.6 on 2019-10-19 11:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0045_add_plural_name_for_log_entry'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reminder',
+            name='jump_url',
+            field=models.CharField(default='invalid', help_text='The jump url to the message that created the reminder', max_length=88),
```

Comment:
Why don't we use an empty string as the value for existing entries? I think that would make the logic at the side of the bot more beautiful:

```py
if jump_url:
    # We know we have a jump_url
```

instead of

```py
if jump_url != "invalid":
    # Do something
```

```suggestion
            field=models.CharField(default='', help_text='The jump url to the message that created the reminder', max_length=88),
```